### PR TITLE
Fix Cloud Build master branch deployment

### LINKED ISSUE #12: Add instructions for starting dev server and manual deployments after #11
@ramyabuva let's add this supporting info to the README when you get a chance.

### PR DESCRIPTION
**Pull request by ramyabuva**
Closes #12

### CODE REVIEW

**Review by savsa — APPROVED** [review]

File: cloudbuildMaster.yaml [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
-   entrypoint: make
+ - name: 'gcr.io/cloud-builders/docker'
+   args: ['build', '-t', 'gcr.io/step-team-145/make', '.']
+ - name: 'gcr.io/step-team-145/make'
    args: ['devprep']
  - name: maven:3-jdk-8
    entrypoint: mvn
-   args: ['package', 'appengine:deploy']
\ No newline at end of file
+   args: ['package', 'appengine:deploy']
```
Comment (wli) [inline cloudbuildMaster.yaml]: out of curiosity - can this be:

```
entrypoint: gcr.io/step-team-145/make
args: ['deploy']
```

That way you only have to maintain the proper arguments in one place.

**Review by wli — APPROVED** [review]

**Review by triplehair — APPROVED** [review]

File: cloudbuildMaster.yaml (hunk 1 above)
Comment (ramyabuva) [reply cloudbuildMaster.yaml]: Before, the docker image didn't support maven, but I just changed the image to another one and this seems to work !!

File: cloudbuildMaster.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
- - name: maven:3-jdk-8
-   entrypoint: mvn
-   args: ['package', 'appengine:deploy']
\ No newline at end of file
+ - name: 'gcr.io/cloud-builders/docker'
+   args: ['build', '-t', 'gcr.io/step-team-145/make', '.']
+ - name: 'gcr.io/step-team-145/make'
+   args: ['deploy']
+
+images:
```
Comment (wli) [inline cloudbuildMaster.yaml]: what does the `images` section mean?

Comment (ramyabuva) [reply cloudbuildMaster.yaml]: it pushes the image to the Google container registry so we can pull it whenever we want to use it

Comment (ramyabuva) [reply cloudbuildMaster.yaml]: Technically I could remove it, I don't think its completely necessary for our purposes

Comment (wli) [reply cloudbuildMaster.yaml]: Oh I see. Can we rename the image to `gcr.io/step-team-145/image` or something? Naming it `make` is quite confusing.

**Comment by wli** [comment]
LGTM

**Pull request merged**